src/PharoThings-Devices-LCD1602.package/PotLCD1602Device.class/instance/clear.st

```diff
@@ -0,0 +1,7 @@
+controlling
+clear
+	self write8: '00000001' mode: 0.
```

Comment:
I thought it is old class. Should it be removed?

Reply:
Yes, please. The package PharoThings-Devices-LCD1602 can be removed, it is from the previous PR.